kong/router.lua

```diff
@@ -371,7 +371,7 @@ do
 
             if m then
               if m.stripped_uri then
-                ctx.matches.stripped_uri = "/" .. m.stripped_uri
+                ctx.matches.stripped_uri = m.stripped_uri
```

Comment:
> The variable should probably be renamed

The name is fine as is, it is still the stripped part of the URL. I was mostly worried about the breaking change; but it might be a necessary evil here.